lib/utils.js

```diff
@@ -55,34 +55,68 @@ exports.splitPop = (input, splitter) => {
 };
 
 /**
- * Gets plugins from node_modules and other directories
+ * Get plugin directories from global config
  *
- * @param {object} globalConfig - Configuration object
+ * @param {object} globalConfig - global configuration file contains plugin directories
  *
- * @returns {object} - all input plugins with path
+ * @returns {array} array of directory paths
  */
-exports.getPlugins = (globalConfig) => {
-  const existing = {};
+exports.getPluginsDirs = (globalConfig) => {
   const root = exec('npm root').toString().trim();
-  let dir = _.get(globalConfig, 'content.plugins.directory', []);
+  const core = _.get(globalConfig, 'content.plugins.core.directory', []);
```

Comment:
removed `core` and `local` as separate configs and plugin sources!

next issue please!